tests/sp2n/Test_2as_base.cc

```diff
@@ -1,13 +1,12 @@
 #include <Grid/Grid.h>
 
-#define verbose 0
+#define verbose 1
 
 using namespace Grid;
 
-int main(int argc, char** argv) {
-  Grid_init(&argc, &argv);
+static void antisymm_base() {
```

Comment:
Mostly code duplication with `symm_base`. We should extract the parts that are actually different into functions and remove the code duplication. I will continue reading through the details after this is fixed (don't want to read it twice).